examples/multisite/example/models.py

```diff
@@ -6,16 +6,11 @@
 
 class AbstractExamplePage(Page):
     body = StreamField(
-        [
-            ('paragraph', blocks.RichTextBlock())
-        ],
+        [("paragraph", blocks.RichTextBlock())],
         use_json_field=True,
     )
 
-    content_panels = [
-        FieldPanel('title'),
-        FieldPanel('body')
-    ]
+    content_panels = [FieldPanel("title"), FieldPanel("body")]
```

Comment:
Same here, I think `content_panels` looks better when it's split on multiple lines.